spec/features/homepage_spec.rb

```diff
@@ -0,0 +1,17 @@
+feature 'Homepage' do
+  scenario 'have correct content' do
+    visit('/')
+    expect(page).to have_content('Rock, Paper, Scissors')
+    expect(page).to have_content('Your name:')
+    expect(page).to have_field('name')
+    
+    expect(page).to have_content('Choose your move:')
+    expect(page).to have_button('Rock')
+    expect(page).to have_button('Paper')
+    expect(page).to have_button('Scissors')
```

Comment:
Asserting on all the content is probably a good idea, I think I just asserted I was on the right page (i.e. the title was correct or whatever) but this is better.